### PR TITLE
fix: Arguments order fix when parsing array parameters

### PR DESCRIPTION
**Pull request by daycry**
Fix: Recorder configuration get params

### CODE REVIEW

**Comment by daycry** [comment]
1) Exception in third-party event subscriber: Library hooks don't exist: ,
#0 C:\laragon\www\mailchimp\vendor\angelov\phpunit-php-vcr\src\Subscribers\ConfigureRecorder.php(43): VCR\Configuration->enableLibraryHooks(Array)
#1 C:\laragon\www\mailchimp\vendor\phpunit\phpunit\src\Event\Dispatcher\DirectDispatcher.php(102): Angelov\PHPUnitPHPVcr\Subscribers\ConfigureRecorder->notify(Object(PHPUnit\Event\TestRunner\ExecutionStarted))
#2 C:\laragon\www\mailchimp\vendor\phpunit\phpunit\src\Event\Dispatcher\DeferringDispatcher.php(45): PHPUnit\Event\DirectDispatcher->dispatch(Object(PHPUnit\Event\TestRunner\ExecutionStarted))       
#3 C:\laragon\www\mailchimp\vendor\phpunit\phpunit\src\Event\Emitter\DispatchingEmitter.php(246): PHPUnit\Event\DeferringDispatcher->dispatch(Object(PHPUnit\Event\TestRunner\ExecutionStarted))       
#4 C:\laragon\www\mailchimp\vendor\phpunit\phpunit\src\TextUI\TestRunner.php(58): PHPUnit\Event\DispatchingEmitter->testRunnerExecutionStarted(Object(PHPUnit\Event\TestSuite\TestSuiteWithName))      
#5 C:\laragon\www\mailchimp\vendor\phpunit\phpunit\src\TextUI\Application.php(198): PHPUnit\TextUI\TestRunner->run(Object(PHPUnit\TextUI\Configuration\Configuration), Object(PHPUnit\Runner\ResultCache\DefaultResultCache), Object(PHPUnit\Framework\TestSuite))

File: src/Extension.php [hunk 1]
```diff
@@ -50,9 +50,12 @@ private function parameterAsArray(ParameterCollection $parameters, string $name)
...
-            fn(string $value): string => trim($value),
-            explode($value, ",")
-        );
+        $data = [];
+        foreach(explode(',', $value) as $item)
+        {
+            $data[] = trim($item);
+        }
+
+        return $data;
```
Comment (angelov) [inline src/Extension.php]: Thank you for the contribution @daycry. Can you please provide some examples where the changed code will produce different output compared to the existing one?

Comment (angelov) [reply src/Extension.php]: Ah, i see the issue now. The order of the `explode` arguments is wrong in the existing code. Can you please update the PR to only fix the order please?

Comment (daycry) [reply src/Extension.php]: Done

**Review by daycry — COMMENTED** [review]
Done

**Comment by angelov** [comment]
Thank you again @daycry. I published a new version with the fix.

**Pull request merged**